python/scripts/add_users_all_org_members_github_team.py

```diff
@@ -1,55 +1,45 @@
+"""
+
+"""
 import os
-import sys
-import time
-import traceback
 
 from python.services.github_service import GithubService
 
+MINISTRYOFJUSTICE_GITHUB_ORGANIZATION_NAME = "ministryofjustice"
+MINISTRYOFJUSTICE_GITHUB_ORGANIZATION_BASE_TEAM_NAME = "all-org-members"  # Contains a base set of permissions for all users in MoJ
 
-def print_stack_trace(message):
-    """Print a stack trace when an exception occurs
-
-    Args:
-        message (string): A message to print when exception occurs
-    """
-    print(message)
-    try:
-        exc_info = sys.exc_info()
-    finally:
-        traceback.print_exception(*exc_info)
-        del exc_info
-
-
-def run(github_service: GithubService):
-    """A function for the main functionality of the script"""
-
-    try:
-        gh = github_service.github_client_core_api
-        org = gh.get_organization("ministryofjustice")
-        team_id = github_service.get_team_id_from_team_name("all-org-members")
-        gh_team = org.get_team(team_id)
-        all_org_members = gh_team.get_members()
-        org_members = org.get_members()
-        for member in org_members:
-            if member not in all_org_members:
-                print(member)
-                gh_team.add_membership(member)
-                # Delay for GH API
-                time.sleep(3)
-    except Exception:
-        message = "Warning: Exception in Run()"
-        print_stack_trace(message)
+MOJ_ANALYTICAL_SERVICES_GITHUB_ORGANIZATION_NAME = "moj-analytical-services"
+MOJ_ANALYTICAL_SERVICES_GITHUB_ORGANIZATION_BASE_TEAM_NAME = "everyone"  # Contains a base set of permissions for all users in MoJAS
 
 
-def main():
-    org_token = os.getenv("ADMIN_GITHUB_TOKEN")
-    if not org_token:
+def get_environment_variables() -> tuple[str, str]:
+    github_token = os.getenv("ADMIN_GITHUB_TOKEN")
+    if not github_token:
         raise ValueError(
             "The env variable ADMIN_GITHUB_TOKEN is empty or missing")
-    github_service = GithubService(org_token, "ministryofjustice")
-    print("Start")
-    run(github_service)
-    print("Finished")
+
+    github_organization_name = os.getenv("GITHUB_ORGANIZATION_NAME")
+    if not github_organization_name:
+        raise ValueError(
+            "The env variable GITHUB_ORGANIZATION is empty or missing")
+
+    return github_token, github_organization_name
```

Comment:
Went for the env var method since it was already established here (and I had a better idea on how to stub them 🧪🙈)

Purposely separated this logic into its own function so we should be able to easily swap out for CLI args in the future if we chose to standardise around that 😁

Reply:
I have no problem with this. It should be "organisation" rather than "organization" unless we intend to sell to our American counterparts.

Reply:
I like sticking with American English - since that's what is used under the hood by everyone - may as well leak it all the way to the developer instead of having to decide where you convert `organisation -> organization` in the layers 🍔

Or maybe we're just too dependent on American code 🇺🇸😂